go-controller/pkg/ovn/master.go

```diff
@@ -85,6 +86,8 @@ func (oc *Controller) Start(kClient kubernetes.Interface, nodeName string, wg *s
 				// run the End-to-end timestamp metric updater only on the
 				// active master node.
 				metrics.StartE2ETimeStampMetricUpdater(oc.stopChan, oc.ovnNBClient)
+				go oc.ensureClusterRaftMembership(util.OvnNbdbLocation)
```

Comment:
Could it happen that we are not the leader anymore and `OnStoppedLeading` gets called? In that case, would it make sense to stop these?

Reply:
If leader stops it will os.exit so the goroutine will get killed anyway. If the master for some reason got stopped but was still leader oc.stopChan would close and that would also stop the goroutine.

Reply:
Got it, so one active and healthy leader cannot be replaced by a new one for any reason. Thanks!